### ISSUE TITLE
How is this project licensed?

### ISSUE THREAD

**Issue by Minoru** [Refefer/propagon#2]
Hello! How is this code licensed? Am I allowed to re-use it in my own project?

Would be nice if it was placed under a free software or open-source license.

**Comment by Refefer** [comment]
Yes, feel free to use it as you'd like.  This is apache2/MIT licensed (though it's clearly missing in the repo).  I'll add the license file tonight.

**Comment by Minoru** [comment]
Excellent, thank you very much!
